modules/nextflow/src/main/groovy/nextflow/extension/ChannelExtensionDelegate.groovy

```diff
@@ -44,10 +43,12 @@ class ChannelExtensionDelegate implements DelegatingPlugin {
 
     private Session getSession() { Global.getSession() as Session }
 
-    final private Map<String, ChannelFactory> channelFactories = new HashMap<>()
-
     final private Map<String,Object> operatorExtensions = new HashMap<>()
 
+    final private Map<String,Tuple2<String,Object>> aliasOperatorExtensions = new HashMap<>()
```

Comment:
Wondering if the two could not be unified into a single `operatorExtensions`?

Reply:
yes, no problem, was to no complicate and keep separate them but we can use only one Map